tests/fixtures/Types/GeoJsonType.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Jsor\Doctrine\PostGIS\Types;
+
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+
+final class GeoJsonType extends GeographyType
```

Comment:
Is this file in the wrong folder?
Would be cool to have this new type added too

Reply:
No, it is meant to be a fixture with respect to this PR. 

Agreed it might be a nice feature, but beyond scope of this PR :smiley_cat:

Reply:
fun fact: we started using it in our own project too, but altered to be a `geometry` rather than a `geography`:
>  `ST_GeomFromGeoJSON(%s)::geometry`

Reply:
Exactly one of the reasons i only provide very basic types. Even those are probably only starting points for your own application specific types 😉

Reply:
yeah, exactly (with the added fun of having Doctrine not discovering the class if you use hypens `-` as type name :facepalm: )

we got an "auto WSG84 converter" type, but that's a different story :-)